src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -27,6 +27,16 @@ const jsonData: CoreWebVitalsPayload = {
 	ttfb: null,
 };
 
+let hasBeenSent = false;
+
+// Can be tested locally or on prod
+const endpoint =
+	window.location.hostname === 'm.code.dev-theguardian.com' ||
+	window.location.hostname === 'localhost' ||
+	window.location.hostname === 'preview.gutools.co.uk'
+		? 'http://performance-events.code.dev-guardianapis.com/core-web-vitals'
+		: 'https://performance-events.guardianapis.com/core-web-vitals';
+
```

Comment:
Did you mean to include these changes? They don't seem related to this PR?

Reply:
They were in before, I just moved them. It means it can be tested locally with no changes